evap/student/templates/student_index_unfinished_evaluations_list.html

```diff
@@ -3,7 +3,7 @@
 <table class="table table-seamless-links table-vertically-aligned table-headerless">
     <tbody>
         {% for evaluation in unfinished_evaluations %}
-            <tr{% if evaluation.state == 'in_evaluation' and not evaluation.voted_for %}
+            <tr{% if evaluation.state == 'in_evaluation' and not evaluation.voted_for and evaluation.vote_end_date >= today %}
```

Comment:
I don't think this will work. We have a specific datetime at which the evaluation ends, and it is not at 0:00, but rather some offset delayed (some setting I think, don't know the specific name at this point). Isn't there some attribute on evaluation that already does this check for you, with the correct computation? I think we should use something like that.

Reply:
the property is called `vote_end_datetime`